defaults/main.yml

```diff
@@ -32,11 +32,12 @@ alpaca_gemini_base_url: http://localhost:8000/gemini
 alpaca_houdini_base_url: http://localhost:8000/houdini
 alpaca_homarus_base_url: http://localhost:8000/homarus
 alpaca_fits_base_url: http://localhost:8000/crayfits
+alpaca_hypercube_base_url: http://localhost:8000/hypercube
 
 alpaca_settings:
   - pid: ca.islandora.alpaca.http.client
     settings:
-      token.value: islandora
+      token.value: "{{ islandora_syn_token }}"
```

Comment:
You don't want to provide a default for this variable?

Reply:
Ah, yes, that's coming from `groupvars/all/passwords.yml`, so I never thought to add a default.  I'll do that now.